Core/Core/AudioVideo/APIMediaComment.swift

```diff
@@ -22,7 +22,7 @@ public struct APIMediaComment: Codable, Equatable {
     let content_type: String
     let display_name: String?
     let media_id: String
-    let media_type: MediaCommentType
+    let media_type: String
```

Comment:
Empty string? or was there some other value that we'll eventually want to understand?

Reply:
I added a comment about it in code. It was actually `video/*`. We default unknown types to `.video` so I figure that's okay. I did go down the path of checking if the raw value contains strings like "audio" or "video" but not sure if worth it or not.